projects/igniteui-angular/src/lib/grids/column.component.ts

```diff
@@ -1108,7 +1108,7 @@ export class IgxColumnComponent implements AfterContentInit {
                 // If nothing is defined yet take any column at first
                 // We use colEnd to know where the column actually ends, because not always it starts where we have it set in columnSizes.
                 columnSizes[col.colStart - 1] = {
-                    field: col.field,
+                    ref: col,
```

Comment:
What is the type of the objects in this array? I see keys being changed, but no interface change.